src/kava.js

```diff
@@ -84,6 +88,22 @@ class Kava extends BasePlugin {
         this._model.updateModel({playerJSLoadTime: entry.duration});
       }
     }
+
+    if (window.PerformanceObserver) {
+      this._performaceObserver = new window.PerformanceObserver(this._handleNewPerformanceEntries.bind(this));
+      this._performaceObserver.observe({entryTypes: ['resource']});
```

Comment:
better move this to `loadMedia` life cycle hook and call disconnect on `reset` so we don't listen while there's no media playing

Reply:
also call disconnect on `destroy`